contracts/account/modules/ERC7579DelayedExecutor.sol

```diff
@@ -370,6 +358,16 @@ abstract contract ERC7579DelayedExecutor is ERC7579Executor {
         return (id, schedule_);
     }
 
+    /// @dev See {ERC7579Executor-canExecute}. Allows anyone to execute an operation if it's {OperationState-Ready}.
+    function canExecute(
+        address /* account */,
+        Mode /* mode */,
+        bytes calldata /* executionCalldata */,
+        bytes32 /* salt */
+    ) public view virtual override returns (bool) {
+        return true;
```

Comment:
This would allow anyone to, for example, reexecute an operation. I'm also realizing that the current ERC7579DelayedExecutor is allowing the account to do that!

This would allow anyone to, for example, reexecute an operation. I'm also realizing that the current ERC7579DelayedExecutor is allowing the account to do that!

The issue of the order of errors was solved in the other PR, wdyt?

Reply:
> I'm also realizing that the current ERC7579DelayedExecutor is allowing the account to do that!

Ah no, false alarm, execution is just not given the state. That means that the current implementation is effectively just `state(account, mode, executionCalldata, salt) == OperationState.Ready` haha, cool.